docs/enroute/Melbourne Centre/PIY.md

```diff
@@ -67,4 +67,4 @@ As per the enroute coordination requirements, voiceless coordination exists betw
 
 ## LEA / IND (Oceanic)
 
-As per Standard coordination procedures (link).
+As per Standard coordination procedures [here](https://sops.vatpac.org/oceanic/Coordination/){target=new}
```

Comment:
Can this be changed to the internal link format `[here](../path/to/page)`

Reply:
For some reason that didn't work. Spat out an error